deploy/kind/deployment/moonlink_deployment.yaml

```diff
@@ -1,3 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Let's make config map a separate file, deployment file could be very length and complicated in the future